x-pack/plugins/index_management/public/application/sections/home/template_list/legacy_templates/template_details/template_details.tsx

```diff
@@ -0,0 +1,327 @@
+/*
```

Comment:
I'm assuming this file was just moved into the `legacy_templates` directory, so only did a light review here.